main.js

```diff
@@ -1,9 +1,9 @@
-const doorImage1 = document.getElementById('door1');
-const doorImage2 = document.getElementById('door2');
-const doorImage3 = document.getElementById('door3');
-const botDoorPath = './images/robot.svg';
-const beachDoorPath = './images/beach.svg';
-const spaceDoorPath = './images/space.svg';
+const doorImage1 = document.getElementById("door1");
+const doorImage2 = document.getElementById("door2");
+const doorImage3 = document.getElementById("door3");
+const botDoorPath = "./images/robot.svg";
```

Comment:
@aoliveiramagalhaes I didn't mean to change from `"` to `'`, my editor formatted this automatically, next time you change this file your editor will revert.

We can have a look at [prettier](http://prettier.io/) to keep code styling consistent across editors.